src/components/App/custom-els/FileDrop/index.ts

```diff
@@ -8,7 +8,7 @@ function firstMatchingItem(list: DataTransferItemList, acceptVal: string): DataT
     return accept.trim().split('/').map(part => part.trim());
   }).filter(acceptParts => acceptParts.length === 2); // Filter invalid values
 
-  return Array.from(list).find((item) => {
+  return Array.from<DataTransferItem>(list).find((item) => {
```

Comment:
I'm not seeing the issue related to this.

Reply:
My linter in Code keeps highlighting at as red and warning, it's not happening in precheck.. I can remove, but it was mega annoying.

Reply:
I can add an issue to track it if needs be, but I was clearing up a warning on my code instance.

Reply:
I don't think this is being flagged by one of the project's linters or typescript. I'm not seeing it here.